### PR TITLE
Ram | Completed Week 1 exercises

### PR DESCRIPTION
**Pull request by Ram-lankada**
1. Finite fields
2. Elliptic curves

### CODE REVIEW

**Review by rajarshimaitra — CHANGES_REQUESTED** [review]
Sorry, I couldn't understand which file is what. 

Can you keep the folder structure same as it is in the pb cohort repo here https://github.com/jimmysong/programmingbitcoin/tree/master/code-ch01?

Do not rename files. Just commit your version of the same files.

**Comment by Ram-lankada** [comment]
Yes, definitely. 
I've named them for my better understanding.

File: exercises/week 1/solutions/Ram/chapter2/ecc.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
+from unittest import TestCase
+
+
+# tag::source1[]
+class FieldElement:
+
+    def __init__(self, num, prime):
```
Comment (rajarshimaitra) [inline exercises/week 1/solutions/Ram/chapter2/ecc.py:7]: It seems you pasted the wrong `ecc.py`? This doesn't have the `Point` class. you need the ecc of ch-02..

**Review by rajarshimaitra — CHANGES_REQUESTED** [review]

File: exercises/week 1/solutions/Ram/chapter2/ecc.py (hunk 1 above)
Comment (Ram-lankada) [reply exercises/week 1/solutions/Ram/chapter2/ecc.py]: Yeah! i've solved it now.

**Review by rajarshimaitra — APPROVED** [review]
ACK b841b8c73003b4a6bc93726d01e5da40090293aa

**Pull request merged**